server/basicServices/basicServices/services/PrepareALTForwardingAutomation.js

```diff
@@ -110,7 +110,7 @@ function getLTPForwardingAutomationInputListAsync(logicalTerminationPointconfigu
              * ServiceRequestCausesLtpUpdateRequest update-ltp
              ************************************************************************************/
             if (logicalTerminationPointconfigurationStatus) {
-                if (logicalTerminationPointconfigurationStatus.hasOwnProperty('httpClientConfigurationStatus')) {
+                if (Object.prototype.hasOwnProperty.call(logicalTerminationPointconfigurationStatus, 'httpClientConfigurationStatus')) {
```

Comment:
using 'in' seems slightly less verbose and should be enough for this usage

Reply:
I removed if statements completely because they are not needed there.